src/data/data.js

```diff
@@ -66,6 +66,15 @@ const pageData = {
         ],
         backgroundImg: '',
       },
+      {
+        component: 'scroll-indicator',
+        image: {
+          src: 'arrow-scroll.png',
+          alt: 'scroll page',
+        },
+        scrollTo: 'scroll',
```

Comment:
If this element will be an id, you could consider using to be the name of the next component.